build.gradle

```diff
@@ -116,11 +116,8 @@ dependencies {
 
 	// Used for converting GraphQL request output to XML:
 	implementation 'com.github.javadev:underscore-lodash:1.26'
-	// For JSON schema validation
-	implementation('io.vertx:vertx-json-schema:4.4.1') {
-		// We don't actually need most of these (dragged in through vertx-core)
-		exclude group: 'io.netty'
-	}
+	// For JSON schema validation, supports detail error reporting
+	implementation 'com.networknt:json-schema-validator:1.0.81'
 	// But we do need io.netty.buffer.ByteBufOutputStream, so explicitly include that here
 	implementation 'io.netty:netty-buffer:4.1.92.Final'
```

Comment:
This dependency can also be removed, since it is only there for vetrx-json-schema.